lib/chat/db/change_tracker/tracking.ex

```diff
@@ -120,4 +114,16 @@ defmodule Chat.Db.ChangeTracker.Tracking do
     state
     |> tracker(next_id: id + 1, keys: new_keys, items: items |> Map.put(id, item))
   end
+
+  defp expiry_callback(changes, :dev) when length(changes) > 0 do
```

Comment:
I do not want this checks to be done in runtime. We can place them in compile time

Also `changes != []` is faster than `length(change)` I guess

Reply:
I understand. Dialyzer will complain, though, that the dev/test part of the code can never get executed.
But I'll change it for now. When we setup Dialyzer in CI we'll need to figure that out.

Reply:
now it is compiled in something like
```
...
defp expiry_callback do
 case :dev do
   :test -> refute
   :dev -> error
   _ -> nil
 end
end
```

The idea is to create a macro that will change function body depending on environment it  is compiled for

Reply:
Ah, now I get it 😃